dashboard.py

```diff
@@ -119,7 +120,20 @@ def time_info(updatedAt):
     return s
 
 def print_dashboard(data):
-    print("<h1>{}</h1>".format(data["title"]))
+    # If there are no PRs, skip the table header and print a bold notice such as
+    # "There are currently **no stale `delegated` PRs. Congratulations!".
+    if not data["output"]:
+        description = {
+            "queue" : "PRs on the review queue",
+            "stale-maintainer-merge" : "stale PRs labelled maintainer merge",
+            "stale-delegated" : "stale delegated PRs",
+            "stale-ready-to-merge" : "stale PRs labelled ready-to-merge",
+            "stale-new-contributor" : "stale PRs by new contributors",
+            "unlabelled" : "unlabelled PRs"
+        }
+        print(f'There are currently <b>no</b> {description[data["id"]]}". Congratulations!\n')
+        return
```

Comment:
I also did not test this locally. In the sense that the dashboard still works, but I did not simulate an empty table. But I'm happy to test this in production.

Reply:
I've now tested this locally, and it doesn't exactly seem to work.
```suggestion
    if not data["output"][0]["data"]["search"]["nodes"]:
        description = {
            "queue" : "PRs on the review queue",
            "stale-maintainer-merge" : "stale PRs labelled maintainer merge",
            "stale-delegated" : "stale delegated PRs",
            "stale-ready-to-merge" : "stale PRs labelled ready-to-merge",
            "stale-new-contributor" : "stale PRs by new contributors",
            "unlabelled" : "unlabelled PRs"
        }
        print("<h1 id=\"{}\">{}</h1>".format(data["id"], data["title"]))
        print(f'There are currently <b>no</b> {description[data["id"]]}. Congratulations!\n')
        return
```

Reply:
Thanks for testing. Applied this fix. Can you re-test, when you have a chance?

Reply:
Looks good to me